.livekit/bootstrap.yaml

```diff
@@ -0,0 +1,7 @@
+target: web
+requires:
+    - pnpm
+install:
+    - "pnpm install"
+dev:
+    - "pnpm dev"
```

Comment:
I'm not sure if the `bootstrap.yaml` approach has been agreed upon, but I think it would make more sense to have the env var mapping in here rather than altering example env files

Reply:
That would work. It's more repetitive for projects with multiple components (say, a `client` and `server` directory written in different languages) but probably overall better.

Reply:
Would it have to be repetitive if the `env` key was a root-level mapping that could be used by different (all) components?

Reply:
I suppose not

Reply:
My new thinking here is that we actually use [task](https://taskfile.dev/) as an embedded task runner, and use their `taskfile.yaml` format out of the box. It comes with pretty awesome features for working with environment and lets you write bash script syntax even for windows targets.